apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/api/Sort.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright © 2018-2021 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.util.api;
+
+/**
+ * @author Andrii Boiarskyi
+ * @see
+ * @since 1.0.0
+ */
+public class Sort {
+    public static final String ASC = "ASC";
+    public static final String DESC = "DESC";
+    private final String order;
+
+    public Sort(String order) {
+        this.order = order;
+        if (!isASC() && !isDESC()) {
+            throw new IllegalArgumentException("Not allowed sort value: " + order);
+        }
+    }
+
+    /**
+     * Returns the Sort object with default values for unknown ordering.
+     * The default value is ASC.
+     *
+     * @param order the given order
+     * @return the Sort object
+     */
+    public static Sort of(String order) {
+        if (ASC.equalsIgnoreCase(order) || DESC.equalsIgnoreCase(order)) {
+            return new Sort(order);
+        } else {
+            return asc();
```

Comment:
Asc is not a default sort format for most of paginated API responses, including blokchain transactions, blocks, account ledger, etc

Reply:
changed to DESC;